### PR TITLE
Update docker settings to expose WS API - Closes #444

### LINKED ISSUE #444: Update docker exposed port and running plugins and configs
### Description

Docker is currently exposing http api endpoint, but we need to re-consider if we should expose for example websocket rpc endpoint.
Also, which plugins, config or flag should be used in docker must be re-considered.

- Documentation needs to be updated (lisk.io / README)

Update default ports of core across network to be
```
P2P: 7667
RPC: 7887
Plugins: 8000 + X
```

### Motivation

With new set of feature, docker setting must be re defined.
We have different defaults per network right now, but
* Hard to remember the port for each network
* No specific reason to change the port per network. As P2P will reject connection based on the network identifier.
* Different default files will be required for docker

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #444 

### How was it solved?

- Add LISK_API_WS_PORT environment variable
- Add docker setting for LISK_API_WS_PORT

### How was it tested?
- Use docker and check if API_WS_PORT is reachable

### CODE REVIEW

File: docker/.env.mainnet [hunk 1]
```diff
@@ -5,3 +5,4 @@ ENV_LISK_VERSION=3.0.0
 # Ports to expose
 ENV_LISK_HTTP_API_PLUGIN_PORT=8000
 ENV_LISK_PORT=8001
+ENV_LISK_API_WS_PORT=8080
```
Comment (fchavant) [inline docker/.env.mainnet:8]: - why is the API WS port not `8002`?
- why is it the same accross networks (while the other ports changed depending on the network)?
-

File: config/devnet/config.json [hunk 2]
```diff
@@ -15,11 +15,11 @@
 			}
 		],
 		"rewards": {
-			"milestones": ["500000000", "400000000", "300000000", "200000000", "100000000"],
+			"milestones": ["766700000", "400000000", "300000000", "200000000", "100000000"],
 			"offset": 2160,
 			"distance": 3000000
 		},
-		"minRemainingBalance": "5000000",
+		"minRemainingBalance": "7667000",
```
Comment (fchavant) [inline config/devnet/config.json]: How do `milestones` and `minRemainingBalance` relate to port numbers?

File: src/commands/start.ts [hunk 3]
```diff
@@ -38,6 +38,10 @@ interface Flags {
 }
 
 const LOG_OPTIONS = ['trace', 'debug', 'info', 'warn', 'error', 'fatal'];
+const DEFAULT_P2P_PORT = 7667;
+const DEFAULT_RPC_PORT = 7887;
+const DEFAULT_PLUGIN_PORT_HTTP = 8000;
+const DEFAULT_PLUGIN_PORT_MONITOR = 8001;
```
Comment (fchavant) [inline src/commands/start.ts:44]: I thought we had agreed it was a good idea to keep the existing ports as they were.
Right now 8001 is the p2p port one has to have open on one's server. After this change it'll be 7667. And 8001 will be something completely different.

**Review by fchavant — APPROVED** [review]

File: docker/.env.betanet [hunk 4]
```diff
@@ -5,3 +5,4 @@ ENV_LISK_VERSION=3.0.0-beta.5
 # Ports to expose
 ENV_LISK_HTTP_API_PLUGIN_PORT=4000
 ENV_LISK_PORT=5000
+ENV_LISK_API_WS_PORT=8080
```
Comment (fchavant) [inline docker/.env.betanet:8]:
```suggestion
ENV_LISK_HTTP_API_PLUGIN_PORT=5000
ENV_LISK_PORT=5001
ENV_LISK_API_WS_PORT=5002
```

File: docker/.env.development [hunk 5]
```diff
@@ -5,3 +5,4 @@ ENV_LISK_VERSION=3.0.0-beta.5
 # Ports to expose
 ENV_LISK_HTTP_API_PLUGIN_PORT=4000
 ENV_LISK_PORT=5000
+ENV_LISK_API_WS_PORT=8080
```
Comment (fchavant) [inline docker/.env.development:8]:
```suggestion
ENV_LISK_PORT=4001
ENV_LISK_API_WS_PORT=4002
```

**Review by fchavant — COMMENTED** [review]
I suggest we keep the ports XXXY consistent across networks.

**Review by fchavant — APPROVED** [review]

File: docker/README.md [hunk 6]
```diff
@@ -46,6 +46,7 @@ services:
     ports:
       - ${ENV_LISK_HTTP_PORT}
       - ${ENV_LISK_WS_PORT}
```
Comment (ManuGowda) [inline docker/README.md:48]: do we use this?

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**